src/sentry/tasks/symbolication.py

```diff
@@ -68,15 +71,17 @@ def should_demote_symbolication(project_id: int) -> bool:
         return False
     elif always_lowpri:
         return True
-    else:
+    elif settings.SENTRY_ENABLE_AUTO_LOW_PRIORITY_QUEUE:
         try:
-            return (
-                settings.SENTRY_ENABLE_AUTO_LOW_PRIORITY_QUEUE
-                and realtime_metrics.is_lpq_project(project_id)
-            )
+            if lpq_projects:
+                return lpq_projects.__contains__(project_id)
```

Comment:
wouldn’t `project_id in lpq_projects` work here?

Reply:
Yeah, that's nicer. Thanks for reviewing!